.github/ISSUE_TEMPLATE/spike_task.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Spike
+about: Issue template for spikes, research and investigation tasks
+labels: 'spike'
```

Comment:
You don't have a `spike` label.
I can give you a script which creates our (my) standard labels, with same colors and descriptions.

Reply:
Yes, please.